pcs/lib/commands/booth.py

```diff
@@ -629,6 +629,19 @@ def _ticket_operation(
     _ensure_live_env(env, booth_env)
 
     if not site_ip:
+        stdout, stderr, return_code = env.cmd_runner().run(
+            [settings.booth_exec, "status"]
+        )
+
+        booth_status = stdout.rstrip()
+        booth_type = booth_status.split(" ")[4].split("=")[1]
+        if booth_type == "arbitrator":
+            raise LibraryError(
+                ReportItem.error(
+                    reports.messages.BoothSiteIpNotSpecifiedOnArbitrator()
+                )
+            )
+
```

Comment:
This solution is a little bit complex and report message is too specific for arbitrator. 
The main issue is that it raises exception `IndexError: list index out of range` in case there is empty output from `booth status` command which may happen if there is no running booth daemon on the arbitrator node or there is no booth resource running on cluster node.

I would suggest to wrap `env.get_cib()` call to `try ... except` block and in case of `LibraryError` raise another `LibraryErrror` with the existing error message `BoothCannotDetermineLocalSiteIp`. 

This change would also require update to the unit tests. Please look at [test_cannot_find_site_ip](https://github.com/ClusterLabs/pcs/blob/f8381657a52615145566454094f3c13a0a3e9666/pcs_test/tier0/lib/commands/test_booth.py#L2311) test and add a new one named `test_cannot_load_cib`.  It will be almost the same as `test_cannot_find_site_ip` but the difference will be in line with  `self.config.runner.cib.load()` call. A non-zero returncode parameter and some stderr parameter need to be added.